### ISSUE TITLE
Update README

### ISSUE THREAD

**Issue by peterdesmet** [trias-project/indicators#50]
... to the reduced scope of indicators (rather than the full pipeline).

**Comment by damianooldoni** [comment]
Short description of indicators added.
Links to the published indicators added as well.
I will update README in the future adding occurrence-based indicators once I have full view about it.

**Comment by peterdesmet** [comment]
Thanks, changed some minor things, looks good. Repo structure (especially) data is still incomplete, so leaving this one open to remind us.

**Comment by damianooldoni** [comment]
Yes, yo uare right. Thanks. Please, not close this issue, as it works nice to remind me the maintenance of the README.

**Comment by damianooldoni** [comment]
Done. The repo structure contains all pipelines.
